lib/tapioca/cli.rb

```diff
@@ -4,104 +4,113 @@
+      type: :string,
+      desc: "Path to the Tapioca configuration file",
+      default: TAPIOCA_CONFIG
     class_option :verbose,
       aliases: ["-V"],
       type: :boolean,
-      default: false,
-      desc: "Verbose output for debugging purposes"
-
-    map T.unsafe(["--version", "-v"] => :__print_version)
+      desc: "Verbose output for debugging purposes",
+      default: false
 
     desc "init", "initializes folder structure"
     def init
       generator = Generators::Init.new(
-        sorbet_config: Config::SORBET_CONFIG,
-        default_postrequire: Config::DEFAULT_POSTREQUIRE,
-        default_command: Config::DEFAULT_COMMAND
+        sorbet_config: SORBET_CONFIG,
+        default_postrequire: DEFAULT_POSTREQUIRE,
+        default_command: DEFAULT_COMMAND
       )
       generator.generate
     end
 
     desc "require", "generate the list of files to be required by tapioca"
+    option :postrequire, type: :string, default: DEFAULT_POSTREQUIRE
     def require
       generator = Generators::Require.new(
-        requires_path: ConfigBuilder.from_options(:require, options).postrequire,
-        sorbet_config_path: Config::SORBET_CONFIG,
-        default_command: Config::DEFAULT_COMMAND
+        requires_path: options[:postrequire],
+        sorbet_config_path: SORBET_CONFIG,
+        default_command: DEFAULT_COMMAND
       )
       Tapioca.silence_warnings do
         generator.generate
       end
     end
 
     desc "todo", "generate the list of unresolved constants"
+    option :todos_path,
+      type: :string,
+      default: DEFAULT_TODOSPATH
+    option :file_header,
+      type: :boolean,
+      desc: "Add a \"This file is generated\" header on top of each generated RBI file",
+      default: true
     def todo
-      current_command = T.must(current_command_chain.first)
-      config = ConfigBuilder.from_options(current_command, options)
       generator = Generators::Todo.new(
-        todos_path: config.todos_path,
-        file_header: config.file_header,
-        default_command: Config::DEFAULT_COMMAND
+        todos_path: options[:todos_path],
+        file_header: options[:file_header],
+        default_command: DEFAULT_COMMAND
       )
       Tapioca.silence_warnings do
         generator.generate
       end
     end
 
     desc "dsl [constant...]", "generate RBIs for dynamic methods"
-    option :generators,
+    option :outdir,
+      aliases: ["--out", "-o"],
+      banner: "directory",
+      desc: "The output directory for generated RBI files",
+      default: DEFAULT_DSLDIR
+    option :file_header,
+      type: :boolean,
+      desc: "Add a \"This file is generated\" header on top of each generated RBI file",
+      default: true
+    option :only,
       type: :array,
-      aliases: ["--gen", "-g"],
       banner: "generator [generator ...]",
-      desc: "Only run supplied DSL generators"
-    option :exclude_generators,
+      desc: "Only run supplied DSL generators",
```

Comment:
Let's be consistent with the options for `gems`

```suggestion
      desc: "Only run supplied DSL generator(s)",
```